MeetupPlatformApi/Features/Authentication/GetCurrentUserInfo/GetCurrentUserInfoFeature.cs

```diff
@@ -19,8 +19,19 @@ public GetCurrentUserInfoFeature(ApplicationContext context, IMapper mapper)
         this.mapper = mapper;
     }
 
+    /// <summary>
+    /// Get current user info.
+    /// </summary>
+    /// <response code="200">Returns current user info.</response>
+    /// <response code="401">If the the user unauthorized.</response>
+    /// <response code="500">If there are database interaction errors.</response>
     [HttpGet("/api/users/me")]
     [Authorize]
+    [ProducesResponseType(typeof(UserInfoDto), StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
```

Comment:
You are using the `SecurityRequirementsOperationFilter`, that automatically adds `401 Unauthorized` and `403 Forbidden` status codes to the endpoint specification.

Reply:
Fixed.